pkgs/applications/networking/cluster/terragrunt/default.nix

```diff
@@ -23,9 +23,11 @@ buildGoModule rec {
   ];
 
   meta = with lib; {
+    homepage = "https://terragrunt.gruntwork.io";
+    changelog = "https://github.com/gruntwork-io/terragrunt/releases/tag/v${version}";
     description = "A thin wrapper for Terraform that supports locking for Terraform state and enforces best practices";
-    homepage = "https://github.com/gruntwork-io/terragrunt/";
     license = licenses.mit;
     maintainers = with maintainers; [ peterhoeg jk ];
+    platforms = platforms.all;
```

Comment:
```suggestion
```

Not required when using buildGoModule. Also all has more systems than buildGoModule supports.